tests/pytest.ini

```diff
@@ -29,6 +29,6 @@ filterwarnings =
     error::pennylane.exceptions.PennyLaneDeprecationWarning
     # Suppress expected AutoGraphWarnings that arise from the tests
     error:AutoGraph will not transform the function:pennylane.capture.autograph.AutoGraphWarning
-addopts = --benchmark-disable
+;addopts = --benchmark-disable
```

Comment:
Is this accidental?

Reply:
Oh that's weird then. Usually I just `pytest` for each test suites and it works. How did you run the tests?